packages/vaadin-virtualizer/src/iron-list.js

```diff
@@ -353,27 +371,31 @@ export const PolymerIronList = Class({
    * @param {boolean} fromTop If the potential reusable items are above the scrolling region.
    */
   _getReusables: function (fromTop) {
-    let ith, offsetContent, physicalItemHeight;
-    const idxs = [];
-    const protectedOffsetContent = this._hiddenContentSize * this._ratio;
-    const virtualStart = this._virtualStart;
-    const virtualEnd = this._virtualEnd;
-    const physicalCount = this._physicalCount;
-    let top = this._physicalTop + this._scrollOffset;
-    const bottom = this._physicalBottom + this._scrollOffset;
-    const scrollTop = this._scrollTop;
-    const scrollBottom = this._scrollBottom;
+    var ith, lastIth, offsetContent, physicalItemHeight;
+    var idxs = [];
+    var protectedOffsetContent = this._hiddenContentSize * this._ratio;
+    var virtualStart = this._virtualStart;
+    var virtualEnd = this._virtualEnd;
+    var physicalCount = this._physicalCount;
+    var top = this._physicalTop + this._scrollOffset;
+    var bottom = this._physicalBottom + this._scrollOffset;
+    // This may be called outside of a scrollHandler, so use last cached position
+    var scrollTop = this._scrollPosition;
+    var scrollBottom = this._scrollBottom;
```

Comment:
Let's use `const` and `let` here, please revert the lines where `var` is used.

Reply:
I'm sure there's room for cleaning up, but I've been generally trying to avoid all modifications to the iron-list source (with the exception of unused function/properties removals)